WordPressKit/WordPressOrgXMLRPCValidator.swift

```diff
@@ -325,7 +325,7 @@ open class WordPressOrgXMLRPCValidator: NSObject {
             guard let data = data,
                 let responseString = String(data: data, encoding: String.Encoding.utf8),
                 let parser = WordPressRSDParser(xmlString: responseString),
-                let endpoint = (try? parser.parsedEndpoint()),
+                let endpoint = (((try? parser.parsedEndpoint()) as String??)),
```

Comment:
I think this line should be:

```swift
let xmlrpc = try? parser.parsedEndpoint(),
```

And the immediate next line should be deleted.  This one:

```swift
let xmlrpc = endpoint,
```